edgexfoundry/app-functions-sdk-go
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings

Signed-off-by: Elizabeth J Lee <elizabeth.j.lee@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) - bootstrap changes for common config
      <link to docs PR>

## Testing Instructions

1. run edgex non-secure stack
2. build app-new-service - `make build`
3. run app-new-service using config provider
4. verify that private configuration is loaded in consul
5. ensure that writable in the private config is modifyable - change log level and see change in the terminal

## New Dependency Instructions (If applicable)
